modules/data-entry/src/main/frontend/src/dataHomepage/LiveTable.jsx

```diff
@@ -18,10 +18,13 @@
 //
 
 import React, { useState } from "react";
-import { Paper, Table, TableHead, TableBody, TableRow, TableCell, TablePagination } from "@material-ui/core";
-import { Card, CardHeader, CardContent, CardActions, Typography, Button, withStyles } from "@material-ui/core";
+import { Fab, Paper, Table, TableHead, TableBody, TableRow, TableCell, TablePagination } from "@material-ui/core";
```

Comment:
`Fab` doesn't appear to be used anymore as we made our own rounded (+) button.
```suggestion
import { Paper, Table, TableHead, TableBody, TableRow, TableCell, TablePagination } from "@material-ui/core";
```